lib/manageiq/providers/redfish/engine.rb

```diff
@@ -13,6 +13,14 @@ def self.vmdb_plugin?
         def self.plugin_name
           _('Redfish Provider')
         end
+
+        def self.init_loggers
+          $redfish_log ||= Vmdb::Loggers.create_logger("redfish.log")
```

Comment:
This looks like it used for operations logging, this would probably be useful to see in evm.log

Reply:
I see it also in the refresher log.  Agreed it should use _log.